src/games/strategy/triplea/ai/proAI/util/ProAttackOptionsUtils.java

```diff
@@ -396,6 +396,18 @@ public void findAttackOptions(final PlayerID player, final List<Territory> myUni
       final Map<Territory, Set<Territory>> landRoutesMap, final List<ProAmphibData> transportMapList,
       final List<Territory> enemyTerritories, final List<Territory> territoriesToCheck,
       final boolean isCheckingEnemyAttacks, final boolean isIgnoringRelationships) {
+    findAttackOptions(player, myUnitTerritories, moveMap, unitMoveMap, transportMoveMap, bombardMap, landRoutesMap,
+        transportMapList, enemyTerritories, new ArrayList<Territory>(), territoriesToCheck, isCheckingEnemyAttacks,
+        isIgnoringRelationships);
+  }
+
+  public void findAttackOptions(final PlayerID player, final List<Territory> myUnitTerritories,
```

Comment:
Keep this as one method rather than splitting it up as two? I'm also unsure why the method would be part of the public API and not private, but re-merging it back to one would make that a moot point.